locales/de.json

```diff
@@ -33,5 +33,6 @@
   "Invalid JSON config file: %s": "Fehlerhafte JSON-Config Datei: %s",
   "Path to JSON config file": "Pfad zur JSON-Config Datei",
   "Show help": "Hilfe anzeigen",
-  "Show version number": "Version anzeigen"
+  "Show version number": "Version anzeigen",
+  "Did you mean %s?": "Meinten Sie %s?"
```

Comment:
Agreed. I'm not 100% sure about the German translation I provided, for your information, the "Sie" means "you", but there's actually two you's in German, there's "Sie" and "du" which I could both use here, "Sie" is really formal, "du" really personal, what do you think?